src/spaces/index.js

```diff
@@ -1,22 +1,11 @@
-import "./xyz-d65.js";
-import "./xyz-d50.js";
-import "./xyz-abs-d65.js";
-import "./lab.js";
-import "./lch.js";
-import "./srgb-linear.js";
-import "./srgb.js";
-import "./hsl.js";
-import "./hwb.js";
-import "./hsv.js";
-import "./p3-linear.js";
-import "./p3.js";
-import "./a98rgb-linear.js";
-import "./a98rgb.js";
-import "./prophoto-linear.js";
-import "./prophoto.js";
-import "./rec2020-linear.js";
-import "./rec2020.js";
-import "./oklab.js";
-import "./oklch.js";
+import ColorSpace from "../space.js";
 
-import "./index-hdr.js";
\ No newline at end of file
+import * as spaces from './index-fn.js';
+
+ColorSpace.register(spaces.XYZ_D65);
```

Comment:
Question: do we need to use `RGBColorSpace.register` instead of `ColorSpace.register` when registering RGB color spaces, or can we use the latter for everything?

Reply:
We can use `ColorSpace.register()` for everything. But I don't recommend listing them out like that, just loop over `spaces`

Reply:
Ha, you can count on me for finding aaaalll the ways to reduce duplication 😆